registry/client/auth/session_test.go

```diff
@@ -41,8 +42,8 @@ func testServerWithAuth(rrm testutil.RequestResponseMap, authenticate string, au
 	wrapper := &testAuthenticationWrapper{
 
 		headers: http.Header(map[string][]string{
-			"Docker-Distribution-API-Version": {"registry/2.0"},
-			"WWW-Authenticate":                {authenticate},
+			"X-API-Version":    {"registry/2.0"},
```

Comment:
`Docker-Distribution-API-Version` is the correct header. Want to add this one as well to make sure its general?